etl/etl_target.py

```diff
@@ -0,0 +1,217 @@
+import pandas
+import psycopg2
+import re
+import spacy
+import numpy
+import os
+from datetime import datetime
+from dotenv import load_dotenv
+from os.path import join, dirname
+
+
+dotenv_path = join(dirname(__file__), ".env")
+load_dotenv(dotenv_path)
+
+
+def connect_and_execute_psql(dbase, query, data):
```

Comment:
Don't need to change now, but in the future it might make sense to remove the parameter dbase since it's not used anywhere

Reply:
👍